Tests/PerformanceTest/SyntaxClassifierPerformanceTests.swift

```diff
@@ -26,17 +26,16 @@ public class SyntaxClassifierPerformanceTests: XCTestCase {
 
   func testClassifierPerformance() throws {
     try XCTSkipIf(ProcessInfo.processInfo.environment["SKIP_LONG_TESTS"] == "1")
-    XCTAssertNoThrow(
-      try {
+    try measureInstructions {
+      do {
```

Comment:
We deliberately only wanted to measure the time to run `classifications` here, not the time to `parse`. Parsing the file is just set-up. Could you make sure that `measureInstructions` covers the same lines of code? 

Same in some other test cases.